package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@holo-host/web-sdk",
+  "version": "0.1.0",
+  "description": "Standard development kit for creating Web UIs that work with Holo Hosting.",
+  "main": "build/index.js",
+  "module": "build/index.js",
+  "browser": "dist/holo_hosting_web_sdk.js",
+  "scripts": {
+    "build": "npm run compile && npm run bundle",
+    "compile": "tsc --esModuleInterop --lib es2015,dom --outDir ./build -p ./src",
+    "bundle": "npx webpack --mode production --output-filename holo_hosting_web_sdk.js",
+    "test": "npm run test-unit",
+    "test-unit": "mocha ./tests/unit/"
+  },
+  "author": "Holo Inc.",
```

Comment:
Holo Ltd.

There is no Holo Inc... FWIW

Reply:
Thanks, I wasn't sure about that.  I will update it.